com.xargon.retrodeck.yml

```diff
@@ -0,0 +1,643 @@
+      sha256: 8ddf57b65fb860416979a3f0640c2ad45ddddbbafa82508ef0a0af3ce7061716
+
+  - name: emulationstation-de
+    buildsystem: cmake-ninja
+    cleanup:
+      - es-app
+      - es-core
+    sources:
+    - type: git
+      url: https://gitlab.com/leonstyhre/emulationstation-de.git
+      branch: c3fc30cfd5c26195024e0da328349d83af067b9e
+
+  # ES-DE - END
+
+  - name: art-book-next
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/share/emulationstation/themes/art-book-next-de/
+      - mv -f * ${FLATPAK_DEST}/share/emulationstation/themes/art-book-next-de/
+    sources:
+      - type: git
+        url: https://github.com/anthonycaccese/art-book-next-de.git
+        branch: 4452f8356819d3cd079549b113fb15edb919a0df
+
+
+  # External manifests start
+
+  # RetroArch - https://github.com/flathub/org.libretro.RetroArch
+
+  - name: retroarch
+    config-opts:
+      - '--enable-dbus'
+    make-args:
+      - GLOBAL_CONFIG_DIR=${FLATPAK_DEST}/etc
+    sources:
+      - type: git
+        url: https://github.com/libretro/RetroArch.git
+        branch: de4e56ecf28a09057734f8624af6d62f3088734f
+      - type: file
+        path: rd-submodules/retroarch/org.libretro.RetroArch.appdata.xml
+      - type: file
+        path: rd-submodules/retroarch/retroarch.cfg
+    post-install:
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/
+      - >-
+        mv ${FLATPAK_DEST}/share/pixmaps/retroarch.svg
+        ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/
+      - rmdir --ignore-fail-on-non-empty ${FLATPAK_DEST}/share/pixmaps/
+      - mkdir -p ${FLATPAK_DEST}/etc
+      - >-
+        sed s:@prefix@:${FLATPAK_DEST}:g retroarch.cfg >
+        ${FLATPAK_DEST}/etc/retroarch.cfg
+      - mkdir -p ${FLATPAK_DEST}/share/appdata
+      - cp org.libretro.RetroArch.appdata.xml ${FLATPAK_DEST}/share/appdata
+    modules:
+      - nvidia-cg-toolkit/nvidia-cg-toolkit-3.1.0013.json
+      - shared-modules/SDL/SDL-1.2.15.json
+      - shared-modules/SDL/SDL_image-1.2.12.json
+      - shared-modules/SDL/SDL_mixer-1.2.12.json
+      - shared-modules/SDL/SDL_net-1.2.8.json
+      - shared-modules/libusb/libusb.json
+      - shared-modules/gudev/gudev.json
+      - libbz2/libbz2-1.0.8.json
+      - libaio/libaio-0.3.112.json
+      - shared-modules/glu/glu-9.json
+      - libdecor/libdecor-0.1.0.json
+  - name: retroarch-filers-video
+    subdir: gfx/video_filters
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/RetroArch.git
+        commit: de4e56ecf28a09057734f8624af6d62f3088734f
+  - name: retroarch-filers-audio
+    subdir: libretro-common/audio/dsp_filters
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/libretro/RetroArch.git
+        commit: de4e56ecf28a09057734f8624af6d62f3088734f
```

Comment:
this is the same repo / revision as above.

Reply:
These are copy-pasted from their original flatpaks actually, however I fixed.